### PR TITLE
Feature middle: main section of the mobile version personal portfolio

### PR DESCRIPTION
**Pull request by kemigabocatherine**
### Mobile version projects section
**In this PR;**

- Added an HTML file with the homepage and project sections

- Added a CSS file with styles for the portfolio

- Added a readme file that describes the project

### CODE REVIEW

**Review by andersonlebon — CHANGES_REQUESTED** [review]
Hi @kemigabocatherine 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] In the README your section `Getting started` should not explain the steps you used while building this project rather it should explain how someone else can run or get the project in his/her local environment. you can include something like:
```
To get the content of this project locally you need to run this command in your terminal:
- `git clone yourlink`
- `cd project name
```
- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: top.html [hunk 1]
```diff
@@ -12,30 +12,135 @@
...
+            </div>
+            <p class="p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="butto" >
+                <button type="button" class="ht">html</button>
+                <button type="button" class="ssc">css</button>
+                <button type="button" class="sj">javaScript</button>
+            </div>
+            <button type="button" class="pro1">See Project</button>
+        </div>    
+        <div class="project2">
```
Comment (andersonlebon) [inline top.html:58]: - [ ] The work projects section should be a `<section>` tag. Kindly let's put all projects in one  `<section>` tag, I know you can do it :clap: :clap: 
- [ ] The  work projects section should use` display:grid`

File: top.html [hunk 2]
```diff
@@ -12,30 +12,135 @@
...
+            <div class="par"><p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+            <p class="connect">LET'S CONNECT</p>
+            <div class="contact">
+                <div><img class="github" src="images/Github@2x.png" alt="Github"></div>
+                <div><img class="linkedin" src="images/Linkedinicon@2x.png" alt="Linkedin"></div>
+                <div><img class="vector" src="images/Vector@2x.png" alt="Vector"></div>
+                <div><img class="twitter" src="images/Twitter@2x.png" alt="Twitter"></div>
+                <div><img class="email" src="images/Mail@2x.png" alt="email"></div>
+            </div>
+            <button class="resume" type="button">Get my resume</button>
```
Comment (andersonlebon) [inline top.html:119]: - [ ] The button ( "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design)
Screenshot from 2021-11-24 23-19-42

File: top.html [hunk 3]
```diff
@@ -12,30 +12,135 @@
...
+                <ul class="ellip"><li>HTML</li></ul>
+            </div>
+            <div class="ellipse3">
+                <img src="images2/Ellipse1(2).png" alt  ="ellipse3">&nbsp;&nbsp;
+                <ul class="ellip"><li>CSS</li></ul>   
+            </div>
+                <div class="frame">
+                    <ul class="skills"><li>Frameworks</li></ul>
+                    <img src="images2/Disabled.png" alt="arrow">
+                </div>
```
Comment (andersonlebon) [inline top.html:136]: - [ ] The technology categories ("Languages", "Frameworks" and "Skills") are in a `<ul>`, each of them in a `<li>`

File: top.html [hunk 4]
```diff
@@ -12,30 +12,135 @@
...
+                <div><ul class="canopy"><li>CANOPY</li></ul></div>
+                <div class="end1"><ul><li>Back End Dev</li></ul></div>
+                <div class="end"><ul><li>2015</li></ul></div>
+            </div>
+            <p class="p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="butto">
+                <button type="button" class="ht">html</button>
+                <button type="button" class="ssc">css</button>
+                <button type="button" class="sj">javaScript</button>
+            </div>
```
Comment (andersonlebon) [inline top.html:105]: The technology in each category( "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`

File: top.html [hunk 5]
```diff
@@ -12,30 +12,135 @@
...
+                <div class="end1"><ul><li>Back End Dev</li></ul></div>
+                <div class="end"><ul><li>2015</li></ul></div>
+            </div>
+            <p class="p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="butto" >
+                <button type="button" class="ht">html</button>
+                <button type="button" class="ssc">css</button>
+                <button type="button" class="sj">javaScript</button>
+            </div>
+            <button type="button" class="pro1">See Project</button>
```
Comment (andersonlebon) [inline top.html:56]: - [ ] The button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design)
Screenshot from 2021-11-24 23-19-42

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🎉**
Hello @kemigabocatherine 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**